.github/workflows/binary.yaml

```
@@ -6,8 +6,8 @@ on:
     - cron: "0 1 * * *" # Triggers the build at 1:00 UTC time
 
 jobs:
-  binary:
-    name: Create
+  create-artifact:
```

I would strongly suggest to keep `binary` as it would require the change to be propagated to the entire pipeline / file. Not the best one, I admit, but that's just a job name used in the rest of the file.
```suggestion
  binary:
```

---

That's a good point about the other references, but maybe it would be ok to just rename those? Or is there some external state that depends on this name?

On the other hand, this PR is about avoiding superfluous runs, so if we chose to rename things this fits better in a separate PR.

---

Left the name to binary for now.